storage/EnvironmentImpl.cc

```diff
@@ -109,4 +109,12 @@ namespace storage
 	return !p || strcmp(p, "yes") == 0;
     }
 
+
+    bool
+    support_btrfs_snapshot_relations()
+    {
+	const char* p = getenv("YAST_BTRFS_SNAPSHOT_RELATIONS");
```

Comment:
Any idea how often this function is called? A couple times per program run? A couple hundred times? More?

If it's called very often, would it make sense to cache this value and check the environment only the first time?

Reply:
Right now once per btrfs snapshot during probing. Given the fact that also e.g. lsattr is called for every subvolume the call of getenv seems insignificant. Also when these snapshot relations are stable the whole function can be removed.

Reply:
OK.